lib/trailblazer/activity/testing.rb

```diff
@@ -49,101 +51,123 @@ def self.def_tasks(*names)
-            ["Start.default"] # start
-          )
-
-          implementation = {
-            "Start.default" => Schema::Implementation::Task(st = Implementing::Start, [Activity::Output(Activity::Right, :success)],        []),
-            :B => Schema::Implementation::Task(b = Implementing.method(:b), [Activity::Output(Activity::Right, :success)],                  []),
-            :D => Schema::Implementation::Task(c = bc, [Activity::Output(Implementing::Success, :success)],                  []),
-            :E => Schema::Implementation::Task(e = Implementing.method(:f), [Activity::Output(Activity::Right, :success)],                  []),
-            "End.success" => Schema::Implementation::Task(_es = Implementing::Success, [Activity::Output(Implementing::Success, :success)], []), # DISCUSS: End has one Output, signal is itself?
-          }
-
-          schema = Inter.(intermediate, implementation)
-
-          @_nested_activity = Activity.new(schema)
-        end
+      def nested_activity
+        return @_nested_activity if defined?(@_nested_activity)
+
+        intermediate = Inter.new(
+          {
+            Inter::TaskRef("Start.default")                 => [Inter::Out(:success, :B)],
+            Inter::TaskRef(:B, more: true)                  => [Inter::Out(:success, :D)],
+            Inter::TaskRef(:D)                              => [Inter::Out(:success, :E)],
+            Inter::TaskRef(:E)                              => [Inter::Out(:success, "End.success")],
+            Inter::TaskRef("End.success", stop_event: true) => [Inter::Out(:success, nil)]
+          },
+          ["End.success"],
+          ["Start.default"] # start
+        )
+
+        implementation = {
+          "Start.default" => Schema::Implementation::Task(
+            Implementing::Start,
+            [Activity::Output(Activity::Right, :success)], []
+          ),
+          :B              => Schema::Implementation::Task(
+            Implementing.method(:b),
+            [Activity::Output(Activity::Right, :success)], []
+          ),
+          :D              => Schema::Implementation::Task(bc, [Activity::Output(Implementing::Success, :success)], []),
+          :E              => Schema::Implementation::Task(
+            Implementing.method(:f),
+            [Activity::Output(Activity::Right, :success)], []
+          ),
+          "End.success"   => Schema::Implementation::Task(Implementing::Success, [Activity::Output(Implementing::Success, :success)], []) # DISCUSS: End has one Output, signal is itself?
+        }
+
+        schema = Inter.(intermediate, implementation)
+
+        @_nested_activity = Activity.new(schema)
+      end
 
-        alias_method :bc, :flat_activity
-        alias_method :bde, :nested_activity
+      alias bc flat_activity
+      alias bde nested_activity
 
-        # Tests {:circuit} and {:outputs} fields so far.
-        def assert_process_for(process, *args)
-          semantics, circuit = args[0..-2], args[-1]
```

Comment:
What is wrong with lines like this?

Reply:
we assigning multiple variables in the same line.